extensions/http-curl/tests/unit/InvokeHTTPTests.cpp

```diff
@@ -47,8 +47,7 @@ TEST_CASE("HTTPTestsWithNoResourceClaimPOST", "[httptest1]") {
 
   std::shared_ptr<core::Processor> logAttribute = std::make_shared<org::apache::nifi::minifi::processors::LogAttribute>("logattribute");
 
-  char format[] = "/tmp/gt.XXXXXX";
-  auto dir = testController.createTempDirectory(format);
+  auto dir = testController.createTempDirectory();
```

Comment:
Maybe I am wrong but the dir variable doesn't seem to be used anywhere in this test.

Reply:
Good catch, thanks!  I have removed the unused variables in https://github.com/apache/nifi-minifi-cpp/pull/1139/commits/d61c4e651a4e3b1c36db8da6f8352ec65e76505a.